Foundation/NSArray.swift

```diff
@@ -687,22 +742,26 @@ open class NSMutableArray : NSArray {
         _storage.replaceSubrange(min..<max, with: [_SwiftValue.store(anObject) as AnyObject])
     }
     
-    public convenience init() {
-        self.init(capacity: 0)
+    public override init() {
+        super.init()
     }
     
     public init(capacity numItems: Int) {
-        super.init(objects: [], count: 0)
+        super.init(objects: nil, count: 0)
 
         if type(of: self) === NSMutableArray.self {
             _storage.reserveCapacity(numItems)
         }
     }
     
-    public required convenience init(objects: UnsafePointer<AnyObject>!, count cnt: Int) {
-        self.init(capacity: cnt)
+    public required init(objects: UnsafePointer<AnyObject>?, count cnt: Int) {
```

Comment:
Small nit: why have `count cnt`? I would just replace `cnt` with `count` throughout the body.

Reply:
Its what the documentation has, but maybe thats autogenerated from some other code?
https://developer.apple.com/documentation/foundation/nsarray/1460096-init

Reply:
Yeah, it's probably autogenerated from header files where the original implementation used cnt. But count is a far better name and is the name of the parameter in swift, and it doesn't change the API, so using count: Int is better for the Swift version IMHO.